src/routes/edit-invoice.tsx

```diff
@@ -146,22 +144,22 @@ export default function ViewInvoice() {
                   </span>
                 </Box>
               </Button>
-              <Button
-                color={isEditMode ? 'error' : 'secondary'}
-                variant="contained"
-                onClick={handleEdit}
-              >
-                {isEditMode ? (
-                  <>
-                    <Icon sx={{ mr: 1 }}>close</Icon>
-                    {t('LABELS.DISCARD')}
-                  </>
-                ) : (
-                  <>
-                    <Icon sx={{ mr: 1 }}>edit</Icon>
-                    {t('LABELS.EDIT')}
-                  </>
-                )}
+              <Button color={isEditMode ? 'error' : 'secondary'}>
```

Comment:
Link inside a button seems strange, I think it's invalid? What you need probably is `<Button component={Link} to=....`. Also, I would rather repeat some code instead of logic :D So my preference would be:

```ts
{isEditMode ? 
  <Button component={Link} to={`/invoice/{id}`} color='error'><Icon>close</Icon>{t...}</Button> : 
  <Button component={Link} to={`/invoice/{id}/edit`} color='secondary'><Icon>edit</Icon>{t...}</Button> : 
}
```